src/components/RespondentsTable/RespondentsTable.tsx

```diff
@@ -0,0 +1,90 @@
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { Svg } from 'components/Svg';
+import { Search } from 'components/Search';
+import { Table } from 'components/Table';
+import { useTimeAgo } from 'hooks';
+import { UserData, users } from 'redux/modules';
+import { useAppDispatch } from 'redux/store';
+import { Row } from 'components/Table';
+import { filterRows } from 'utils/filterRows';
+import { StyledFlexAllCenter } from 'styles/styledComponents/Flex';
+
+import { RespondentsTableHeader } from './RespondentsTable.styles';
+import { headCells } from './RespondentsTable.const';
+
+export const RespondentsTable = (): JSX.Element => {
+  const dispatch = useAppDispatch();
+  const { t } = useTranslation('app');
+  const timeAgo = useTimeAgo();
+  const usersData = users.useUserData();
+  const [searchValue, setSearchValue] = useState('');
```

Comment:
Let move this search logic to custom hook

Reply:
Ok, in separate task